pkg/daemon/ceph/client/pool.go

```diff
@@ -696,9 +696,9 @@ func createReplicationCrushRule(context *clusterd.Context, clusterInfo *ClusterI
 		args = append(args, deviceClass)
 	}
 
-	_, err := NewCephCommand(context, clusterInfo, args).Run()
+	output, err := NewCephCommand(context, clusterInfo, args).Run()
 	if err != nil {
-		return errors.Wrapf(err, "failed to create crush rule %s", ruleName)
+		return errors.Wrapf(err, "failed to create crush rule %s. %s", ruleName, string(output))
```

Comment:
"failed to create crush rule on the pool %s %s. %, ruleName, string(output), poolname"

Reply:
Adding the pool name to the error message is not necessary in this case because the callers will wrap with another error message that has the pool name.